### DISCUSSION TITLE
i cant find lluminate\Foundation\Auth\RegistersUsers

### DISCUSSION THREAD

**Discussion by mrhili** [laravel/framework#32869] [General] [answered]
"php": "^7.2.5",
        "acaronlex/laravel-calendar": "^1.0",
        "arcanedev/log-viewer": "7.x",
        "fideloper/proxy": "^4.2",
        "fruitcake/laravel-cors": "^1.0",
        "guzzlehttp/guzzle": "^6.3",
        "laravel/framework": "^7.0",
        "laravel/tinker": "^2.0",
        "laravel/ui": "^2.0",
        "laravelcollective/html": "^6.1",
        "silber/bouncer": "v1.0.0-rc.8",
        "venturecraft/revisionable": "^1.34"
    },
    "require-dev": {
        "barryvdh/laravel-debugbar": "^3.3",
        "facade/ignition": "^2.0",
        "fzaninotto/faker": "^1.9.1",
        "mockery/mockery": "^1.3.1",
        "nunomaduro/collision": "^4.1",
        "phpunit/phpunit": "^8.5"
    },

**Comment by devcircus** [comment, marked as answer]
It's in laravel/ui under the same namespace as it previously was.

**Reply by mrhili** [reply]
I will see in laravel ui

**Reply by mrhili** [reply]
I found it thank you a lot: I didn't know

**Reply by JacobDev37** [reply]
Thanks man, you saved my life
